### PR TITLE
Add some missing keywords + fix Exit Function

### PR DESCRIPTION
**Pull request by stepanhakobyan**
Add some missing keywords.

Fix "Exit Function" and "End" when there was tab before the keyword.

### CODE REVIEW

File: syntaxes/vba.tmGrammar.yml [hunk 1]
```diff
@@ -34,9 +35,9 @@ repository:
       - name: keyword.conditional.end.vba
         match: (?i:\b(End( )?If|End (Select|With)|Next|Wend|Loop(( While)|( Until))?|Exit (For|Do|While))\b)
       - name: keyword.control.vba
-        match: (?i:\b((?<= )E(nd|xit)|As|And|By(Ref|Val)|Goto|Is|Like|Mod|Not|On Error|Optional|Or|Resume Next|Stop|Xor)\b)
+        match: (?i:(\b(Exit (Function|Property|Sub)|As|And|By(Ref|Val)|Goto|Is|Like|Mod|Not|On Error|Optional|Or|Resume Next|Stop|Xor|Eqv|Imp|TypeOf|AddressOf)\b)|(\b(End)\b(?=\n)))
```
Comment (serkonda7) [inline syntaxes/vba.tmGrammar.yml:38]: Good catch with the tab issue.

**Review by serkonda7 — CHANGES_REQUESTED** [review]
Could you please add a link to the documentation of the newly added keywords.

File: syntaxes/vba.tmGrammar.yml (hunk 1 above)
Comment (stepanhakobyan) [reply syntaxes/vba.tmGrammar.yml:38]: @serkonda7 Where should I add the links? In this PR?

Comment (serkonda7) [reply syntaxes/vba.tmGrammar.yml:38]: @stepanhakobyan yes, just post a comment here in this pull request (e.g. reply to this discussion).

**Comment by stepanhakobyan** [comment]
Exit
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/exit-statement

Eqv
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/eqv-operator

Imp
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/imp-operator

TypeOf
https://learn.microsoft.com/en-us/office/vba/Language/Reference/User-Interface-Help/ifthenelse-statement

AddressOf
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/addressof-operator

End
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/end-statement

Event
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/event-statement

RaiseEvent
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/raiseevent-statement

Declare, PtrSafe
https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/declare-statement

WithEvents - Could find documentation on Microsoft's Docs
https://www.oreilly.com/library/view/vb-vba/1565923588/1565923588_ch07-2343-fm2xml.html

**Comment by serkonda7** [comment]
Awesome, thanks. And I found a link for WithEvents in VBA here: https://learn.microsoft.com/en-us/office/vba/excel/Concepts/Events-WorksheetFunctions-Shapes/using-events-with-the-application-object

However only the VB.Net language reference contains an entry about this.
@h8nor do you maybe have some better information here?

**Review by serkonda7 — APPROVED** [review]

**Comment by h8nor** [comment]
Yes, we can use `WithEvents` to refer on an objects to get it methods in the VBA classes. The documentation is incomplete.

**Comment by serkonda7** [comment]
Thank you all 🎉 
Great work and communication

**Comment by h8nor** [comment]
**Note:** The type `Any` doesn't participate in the VBA structure. It works only in the VB as a pointer to the raw unmodified binary data in [Late Binding](https://learn.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/early-late-binding/?view=netframework-4.7.2) (to declare a DLL function/procedure).

**Comment by serkonda7** [comment]
Actually it isn't a VBA type but rather a keyword ([some stackoverflow answer](https://stackoverflow.com/questions/25316973/why-cant-i-use-the-any-as-a-name-in-vba/25317051#25317051)), so the current kind of highlighting may be wrong.
I would be fine tho with keeping as is.

**Comment by h8nor** [comment]
> [`Any` as a reserved word](https://support.microsoft.com/en-us/office/learn-about-access-reserved-words-and-symbols-ae9d9ada-3255-4b12-91a9-f855bdd9c5a2)

**Pull request merged**